packages/protocol/script/config_dcap_sgx_verifier.sh

```diff
@@ -0,0 +1,20 @@
+# for foundry test only!
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
+ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1 \
+LOG_LEVEL=DEBUG \
+REPORT_GAS=true \
+SGX_VERIFIER_ADDRESS=0x1fA02b2d6A771842690194Cf62D91bdd92BfE28d \
+TIMELOCK_ADDRESS=0xB2b580ce436E6F77A5713D80887e14788Ef49c9A \
+ATTESTATION_ADDRESS=0xC9a43158891282A2B1475592D5719c001986Aaec \
+TCB_INFO_PATH=/test/automata-attestation/assets/0923/tcbInfo.json \
+QEID_PATH=/test/automata-attestation/assets/0923/identity.json \
+V3_QUOTE_PATH=/test/automata-attestation/assets/0923/v3quote.json \
+MR_ENCLAVE=0xdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef \
+MR_SIGNER=0xdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef \
+forge script script/SetDcapParams.s.sol:SetDcapParams \
+    --fork-url http://localhost:8545 \
+    --broadcast \
+    --ffi \
+    -vvvv \
+    --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \ #foundry test key
```

Comment:
This private key always needs to match `PRIVATE_KEY`? If so, can we reuse `PRIVATE_KEY` here?

Reply:
@smtmfft

Reply:
Not that familiar with foundry, so it follows another script, will double check.

Reply:
Generally this 2 keys can be different to make each function (could) have its own sender rather than forge test trigger. Here for testing they are the same. So, no problem here.